Makefile

```diff
@@ -16,5 +25,11 @@ build-test:
 	docker build -t openslides-auth-test -f Dockerfile.test .
```

Comment:
I searched for `Dockerfile.test` and `openslides-auth-test` and this line is the only result -> can the Dockerfile.test be removed (I think we discussed it before my vacation)?